toolchain/check/type_structure.h

```diff
@@ -77,24 +77,40 @@ class TypeStructure : public Printable<TypeStructure> {
  private:
   friend class TypeStructureBuilder;
 
+  // Elements of the type structure, indicating the present of a concrete or
+  // symbolic element, and for aggregate concrete types (such as generic types),
+  // nesting for the types inside.
   enum class Structural : uint8_t {
+    // A concrete element in the type structure, such as `bool`.
     Concrete,
+    // A concrete element in the type structure that contains nested types
```

Comment:
```suggestion

    // A concrete element in the type structure that contains nested types
```

Suggesting whitespace between entries, just for readability.